pbcommand/schemas/pipeline_presets.avsc

```diff
@@ -0,0 +1,41 @@
+{
+  "namespace": "com.pacbio.common.models.pipeline_presets",
+  "type": "record",
+  "name": "PipelinePresets",
```

Comment:
Minor change, would you mind changing the name to `PipelinePreset`? Looks great otherwise. Thanks for the tests as well.